src/tv2-common/parts/effekt.ts

```diff
@@ -93,30 +91,27 @@ export function CreateEffektForPartInner<
 	},
 	label: string
 ): Pick<IBlueprintPart, 'autoNext' | 'inTransition'> | false {
-	if (!config.showStyle.BreakerConfig) {
-		context.notifyUserWarning(`Jingles have not been configured`)
+	if (!context.config.showStyle.BreakerConfig) {
+		context.core.notifyUserWarning(`Jingles have not been configured`)
```

Comment:
I know this is very small and out of scope. But in case you come by it, could we change it to `Jingles have not been configured in the Breaker configuration` or something like that? Without someone telling me where it is, i had no idea. But that's of course just for the inexperienced :-)

Reply:
That code was dead. Removed